### PR TITLE
MULE-18269: XML SDK primary parameters are sent as null

### PR DESCRIPTION
**Pull request by elrodro83**

### CODE REVIEW

File: modules/extensions-xml-support/src/main/java/org/mule/runtime/extension/internal/processor/ModuleOperationMessageProcessor.java [hunk 1]
```diff
@@ -164,6 +165,8 @@ public ModuleOperationMessageProcessor(Map<String, String> properties,
       if (parameters.containsKey(parameterName)) {
         final String xmlValue = parameters.get(parameterName).trim();
         result.put(parameterName, new Pair<>(xmlValue, parameterModel.getType()));
+      } else if (PRIMARY_CONTENT.equals(parameterModel.getRole())) {
```
Comment (marianogonzalez) [inline modules/extensions-xml-support/src/main/java/org/mule/runtime/extension/internal/processor/ModuleOperationMessageProcessor.java:168]: this actually applies to any ooptional parameter, not just the primary content ones. Also, if that default value is an expression, where is that evaluated?

Comment (elrodro83) [reply modules/extensions-xml-support/src/main/java/org/mule/runtime/extension/internal/processor/ModuleOperationMessageProcessor.java:168]: the execution evaluates that, will check for other optional params

Comment (elrodro83) [reply modules/extensions-xml-support/src/main/java/org/mule/runtime/extension/internal/processor/ModuleOperationMessageProcessor.java:168]: optional parameters have tests and are green: `ModuleSimpleTestCase#testSetPayloadParamDefaultUseOptionalFlow`

**Comment by elrodro83** [comment]
--validate

**Comment by elrodro83** [comment]
--validate

**Pull request merged**